src/amo/components/Errors/ErrorComponent/index.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+import NestedStatus from 'react-nested-status';
+
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+type Props = {|
+  children: React.Node,
+  className?: string,
+  code: number,
```

Comment:
You have defined a default value for `code` below, via `code={code || 400}`, but `code` is mandatory here, which means it must be passed in. If we really want to allow a client to not pass a `code` then this should be defined as optional.